patching/src/main/java/org/jboss/as/patching/Constants.java

```diff
@@ -43,6 +43,8 @@
     public static final String UNKNOWN = "Unknown";
 
     public static final String AGEOUT_HISTORY = "ageout-history";
+    public static final String AGEDOUT = "agedout";
```

Comment:
@jfdenise You asked on #2699 about this. Yes, this should be "aged-out".

Reply:
Thank-you, that has been updated.